lua/diagnosticls-nvim/formatters/eslint.lua

```diff
@@ -0,0 +1,15 @@
+local fs = require 'diagnosticls-nvim.fs'
+
+return {
+  sourceName = 'eslint',
+  command = fs.get_executable('eslint', 'node'),
+  args = { '--fix', '--fix-to-stdout', '--stdin' '--stdin-filename', '%filepath' },
+  rootPatterns = {
+    '.eslintrc',
+    '.eslintrc.cjs',
+    '.eslintrc.js',
+    '.eslintrc.json',
+    '.eslintrc.yaml',
+    '.eslintrc.yml',
+  },
+}
```

Comment:
I'd rather this be another PR instead, to add the linter, and have this PR focus on the default configs